hw_diag/utilities/quectel.py

```diff
@@ -73,28 +82,35 @@ def find_eg25g_modem() -> Modem:
 @retry(Exception, tries=3, delay=5, backoff=2, logger=logging)
 def download_and_extract(url: str, file_path: str, file_hash: str) -> None:
     download_with_resume(url, file_path, file_hash)
-    untar_cmd = ['tar', '-xzf', file_path, '-C', FW_STORE_PATH]
-    tar_output = ''
+    untar_cmd = ["tar", "-xzf", file_path, "-C", FW_STORE_PATH]
```

Comment:
@kashifpk what do you think about [tarfile](https://docs.python.org/3/library/tarfile.html) instead of shelling out?

@pritamghanghas not a blocker for me

Reply:
Yes @marvinmarnold I'd second using tarfile module instead of shelling out. We all know how @vpetersson feels about shelling out instead of using available libs :-)